src/interactive-window/commands/commandRegistry.ts

```diff
@@ -479,6 +482,15 @@ export class CommandRegistry implements IDisposable {
         }
     }
 
+    private async createNewNotebook(): Promise<void> {
+        this.appShell
+            .showInformationMessage(
+                'This command has been deprecated and will eventually be removed, please use "Create: New Jupyter Notebook" instead'
```

Comment:
trying to see if there's a smart link that I can use to guide them right to the keyboard shortcut